Makefile

```diff
@@ -13,9 +12,10 @@ export TREE?=$(ROOT_DIR)/packages
 BUILD_ARGS?=--pull --no-spinner --only-target-package
 
 VALIDATE_OPTIONS?=-s
-ARCH?=amd64
-REPO_CACHE?=raccos/$(ARCH)
-FINAL_REPO?=raccos/releases-$(ARCH)
+FLAVOR?=opensuse
+DESTINATION?=$(ROOT_DIR)/build
```

Comment:
JFYI I'd would have made the this line like
```
DESTINATION?=$(ROOT_DIR)/build-$(FLAVOR)
```
So I could easily build the iso from opensuse or fedora without having to rebuild the repository. However the geniso extension is dependent on the  `$(ROOT_DIR)/build` path. No big deal, but something to keep in mind for #56

Reply:
Is geniso using it explictly or it's caused by https://github.com/mudler/cOS/blob/master/iso/cOS.yaml#L20 ?

Reply:
Oh, I guess you are right, I did not look into that. I saw it was failing in one of the geniso scripts but I did not check where the path was defined.